datadog/dogshell/monitor.py

```diff
@@ -178,6 +178,8 @@ def _file_post(cls, args):
             name=monitor["name"],
             message=monitor["message"],
             options=monitor["options"],
+            tags=monitor["tags"],
```

Comment:
```suggestion
            tags=monitor.get(["tags"], None)
```
This would raise an exception if user's omit `tags`. Same with `priority`